mappings/7dd58c75bde4b35dc6ef0c8ff5298f3bfba3d0d7c41f178d29ec2b7543726f7764737472696b65.json

```diff
@@ -0,0 +1 @@
lFEzn3iJDFr7UbWgxHxYJv55ewNf/Hwtm23nanbX+5+5M6958wc/O0yNR8fVC/aNzAuVeK7/d7PNZnnRWkyIg7lkYNfHI84ozn26MG7BubPe/MzvkDb391Eu797bPho/0h397JTHAyNp4zox6a7vhIRb9v90osWd6Z1Lo7Mb25ruaREnVujPCmlb83aLbUe6Efc2UT9Qonhz8+/dnJX7PpHj25up8346cyn3nDpd3sT9ru+557ewZnndj7ayfyXk9t6/bFLRzjQPfKrx3L6pzPLKDeYMo7OOPaM3rtrbf+mF3l7xpOv2+/1Dx061h+Tr2On3/3bXpn6dCmsT3sQPn/T9gf3XXbhu/rTp42acaxf675zuu0prQJmRLurf+RzTZnx5tPt39fqSLf/6ETF4hvPWCdZgDH3yLpz5h+ZMf2sJz7WHD16dNHm+74hajl8PalrY+DBeRcubWrz+M2Mbj1y7Jx779+X45SeEk7Ftu9YMXX2cD6pZkkz3DnybMzkBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAE/wfwH4sZ8sIjZkSgAAAABJRU5ErkJggg==", "signatures": [{"signature": "e4c2946dabfb732e766747a3c499d238177a0b07c6075e9517902ca1f69584a37d97f627e1d63f2a9d6fb36b7f5df5602202b16e4a9b40a782fdcc806e49cf01", "publicKey": "a341242846896967fa0d820da28354c49945174f9d66ad20f23d6f1da1aee995"}]}, "description": {"sequenceNumber": 0, "value": "Microsoft BSOD meme", "signatures": [{"signature": "465d089d4b892f487075da7727bbff02958f29e7fb633a737efb6a93f0fe1bdb183d72f40a8048a0cf798f859414b7d53bd111708fbd3ff6859e98f888359409", "publicKey": "a341242846896967fa0d820da28354c49945174f9d66ad20f23d6f1da1aee995"}]}}
```

Comment:
Hi @CardanoTokenIO,

please change the logo. CrowdStrike logo can't be accepted.

Thanks!